lib/tasks/mastodon.rake

```diff
@@ -45,7 +45,7 @@ namespace :mastodon do
         env[key] = SecureRandom.alphanumeric(32)
       end
 
-      vapid_key = Webpush.generate_key
+      vapid_key = LegacyWebpush.generate_key
```

Comment:
Does this method actually change between the two versions?

Reply:
They are the same, and I kept the old one, but I can use the new one